jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/telemetry/CodeWhispererTelemetryService.kt

```diff
@@ -467,6 +472,11 @@ class CodeWhispererTelemetryService {
         )
     }
 
+    fun sendOnboardingClickEvent(language: CodeWhispererProgrammingLanguage, taskType: CodewhispererGettingStartedTask) {
+        // Project instance is not needed. We look at these metrics for each clientId.
+        CodewhispererTelemetry.onboardingClick(null as Project?, language.toTelemetryType(), taskType)
```

Comment:
Does this need a typecast? Can we send it as `null`

Reply:
it does, otherwise it will throw this compile error:
```
[OVERLOAD_RESOLUTION_AMBIGUITY] Overload resolution ambiguity. All these functions match.
public final fun onboardingClick(project: Project?, codewhispererLanguage: CodewhispererLanguage, codewhispererGettingStartedTask: CodewhispererGettingStartedTask, passive: Boolean = ..., value: Double = ..., createTime: Instant = ...): Unit defined in software.aws.toolkits.telemetry.CodewhispererTelemetry
public final fun onboardingClick(connectionSettings: ConnectionSettings? = ..., codewhispererLanguage: CodewhispererLanguage, codewhispererGettingStartedTask: CodewhispererGettingStartedTask, passive: Boolean = ..., value: Double = ..., createTime: Instant = ...): Unit defined in software.aws.toolkits.telemetry.CodewhispererTelemetry
```

Reply:
Does this work: `onboardingClick(project = null, .....)`?

Reply:
it works, will change in the next PR.